tests/rptest/util.py

```diff
@@ -200,15 +200,16 @@ def segments_removed():
                 if s.name not in current_segs_names:
                     removed_segments += 1
 
+            redpanda.logger.info(f"removed: {removed_segments}")
             if removed_segments < n:
                 return False
 
         return True
 
     wait_until(segments_removed,
-               timeout_sec=180,
```

Comment:
Why is it correct to decrease this timeout?  I suspect some of the failures in other tests may be the result of this change.

Reply:
Nice catch, this wasn't meant to be committed